src/popup/router/components/Header.vue

```diff
@@ -17,7 +17,7 @@
       <ButtonPlain
         v-if="showBack"
         class="icon-btn back"
-        @click="back"
+        @click="$router.back()"
```

Comment:
In case user fisrtly open the page with a backButton enabled, by clicking on it he would end up closing the app.